test/library/packages/UnitTest/Launcher.chpl

```diff
@@ -179,7 +195,7 @@ module Launcher {
     if erroredTestNames.size != 0 then erroredTestNamesStr = erroredTestNames: string;
     if testsPassed.size != 0 then passedTestStr = testsPassed: string;
     if skippedTestNames.size != 0 then skippedTestNamesStr = skippedTestNames: string;
-    var exec = spawn(["./"+executable, "-nl", reqNumLocales: string, "--testNames", 
+    var exec = spawn(["./"+executable, "--numLocales", reqNumLocales: string, "--testNames", 
```

Comment:
What is the impact of this change? or is it just stylistic?

Reply:
I thought `-nl` doesn't work when `comm=none`

Reply:
Have reverted to previous version.